package.json

```diff
@@ -36,6 +36,7 @@
     "@angular/platform-browser": "^7.1.1",
     "@angular/platform-browser-dynamic": "^7.1.1",
     "@angular/router": "^7.1.1",
+    "@blackbaud/auth-client": "^2.12.0",
```

Comment:
This is the primary change. All other changes are simple package version upgrades.